src/network/taraxa_capability.hpp

```diff
@@ -86,7 +86,8 @@ class TaraxaPeer : public boost::noncopyable {
 
   void setAlive() { alive_check_count_ = 0; }
 
-  bool passed_initial_ = false;
+  bool received_initial_status_ = false;
+  bool sent_initial_status_ = false;
```

Comment:
Looks like don't need sent_initial_status_. I think when node received_initial_status_ = true, that also means peer has send initial status.
I may wrong.